RoboFile.php

```diff
@@ -618,4 +625,34 @@ public function writeReleaseVersion($version) {
       ->to(sprintf("'version' => '%s',", $version))
       ->run();
   }
+
+  public function jiraReleaseVersion($opts = ['free' => null, 'premium' => null]) {
+    require_once './tasks/release/Jira.php';
+    if (empty($opts['free']) && empty($opts['premium'])) {
+      $this->yell('No Free or Premium version specified', 40, 'red');
+      exit(1);
+    }
+    $output = [];
+    if (!empty($opts['free'])) {
+      $this->validateVersion($opts['free']);
```

Comment:
Not sure if we have to validate the version here since the Jira API will return an error for non-existing version anyway.

Reply:
We validate the version format here (MAJOR.MINOR.PATCH), not its existence.

Reply:
We create the version name in Jira but there is no validation :( So it is probably good to have validation here, but still why not do it in all in Controller's check version method?

Reply:
It was already used by another command (`writeReleaseVersion`) so I've just reused it.

Reply:
Maybe we could (in the future) also check if version exists within `writeReleaseVersion`